pkg/kinflate/util/util.go

```diff
@@ -124,6 +117,21 @@ func Encode(in types.KObject) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+// EncodeFromKObject encodes the map `in` and output the encoded objects separated by `---`.
+func EncodeFromKObject(in types.KObject) ([]byte, error) {
+	gvknList := []types.GroupVersionKindName{}
+	for gvkn := range in {
+		gvknList = append(gvknList, gvkn)
+	}
+	sort.Sort(types.ByGVKN(gvknList))
```

Comment:
This is doing more than encoding. If it is called only from one place, I will move it to that package instead of having it in utils here.